### PR TITLE
Add SPQR docs

### PR DESCRIPTION
**Pull request by Denchick**

### CODE REVIEW

File: config-example/2shardproxy.yaml [hunk 1]
```diff
@@ -18,16 +18,12 @@ frontend_rules:
...
-    pool_rollback: true
     pool_prepared_statement: true
     auth_rule:
       auth_method: ok
       password: strong
   - pool_mode: TRANSACTION
-    pool_discard: false
-    pool_rollback: true
-    pool_default: true
+    use_by_default: true
```
Comment (reshke) [inline config-example/2shardproxy.yaml:26]: pool_default

File: README.md [hunk 2]
```diff
@@ -2,142 +2,42 @@
...
+# Stateless Postgres Query Router
 
-SPQR is a liquid OLTP sharding for PostgreSQL.
+SPQR is a system for horizontal scaling of PostgreSQL via sharding. We appreciate any kind of feedback and contribution to the project.
+
+For more about SPQR, please see [docs/](docs/).
+
+## Main features
+
+- Transaction and session pooling
```
Comment (Denchick) [inline README.md:14]: Not sure what to write there

File: README.md [hunk 3]
```diff
@@ -2,142 +2,42 @@
...
 ```
-make run
-
-.......
-
-
-Configute routing rules
-
-
-root@spqr_client:/go# connect_adm.sh
```
Comment (Denchick) [inline README.md:28]: Not sure that I should delete it
But it takes a lot of space

File: config-example/1shardproxy.toml [hunk 4]
```diff
@@ -0,0 +1,39 @@
+log_level = "DEBUG5"
```
Comment (Denchick) [inline config-example/1shardproxy.toml:1]: Add toml and json configs just to show that it is possible to use difference format

File: docs/Router.md [hunk 5]
```diff
@@ -0,0 +1,83 @@
...
+
+## shards
+
+| **Name** | **Description**                                                                    |
+| -------- | ---------------------------------------------------------------------------------- |
+| `db`     | the database to connect to                                                         |
+| `usr`    | the username with to connect                                                       |
+| `pwd`    | the username's password                                                            |
+| `hosts`  | list of data shard hosts in `host:port` format                                     |
+| `type`   | can be `DATA` or `WORLD`                                                           |
```
Comment (Denchick) [inline docs/Router.md:74]: Add link to Worldshard.md

File: README.md [hunk 6]
```diff
@@ -2,142 +2,42 @@
...
-ROUTER NOTICE: rerouting your connection
-ROUTER NOTICE: failed to match shard
-ROUTER NOTICE: initialize single shard server conn
-ROUTER NOTICE: adding shard w1
-ROUTER NOTICE: you are receiving message from mock world shard
- worldmock 
------------
- row1
-(1 row)
+SPQR does not have unit tests yet but has end-to-end tests. These tests requires Docker and can be run using `make run`. Also, there are stress tests but it work in progress. For more information on testing, please see `test` and `stress` section in [Makefile](./Makefile).
```
Comment (x4m) [inline README.md:39]: There's regression tests.

**Pull request merged**